src/app/events/events.page.ts

```diff
@@ -8,6 +8,9 @@ import { Event } from 'src/shared/interfaces/Event';
   styleUrls: ['./events.page.scss'],
 })
 export class EventsPage implements OnInit {
+  headerTitle: string = 'Събития';
+  defaultHref: string = '/tabs/home';
+  backButton: boolean = true;
```

Comment:
If called from the tabs - there should be no back button, only id called with filter from the home page